lib/routes/account.js

```diff
@@ -447,15 +515,19 @@ module.exports = function (
           }
         }
 
-        function sendNewDeviceLoginNotification () {
-          if (config.newLoginNotificationEnabled && wantsKeys(request)) {
+        function sendVerifyLoginEmail() {
+          if (wantsKeys(request) && doSigninConfirmation) {
```

Comment:
What happens is `wantsKeys` is false, but `doSigninConfirmation` is true?  Do we wind up with an unverified token for which we never send an email?  I guess that's what we want but it could be easy for a casual reader to think this was a bug, may be worth a comment somewhere here to explain the situation.